content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -414,6 +450,14 @@ Used on: IngressClass
 
 When a single IngressClass resource has this annotation set to `"true"`, new Ingress resource without a class specified will be assigned this default class.
 
+### ingress.kubernetes.io/whitelist-source-range
+
+Example: `ingress.kubernetes.io/whitelist-source-range: "10.0.0.0/24,172.10.0.1"`
+
+Used on: Ingress
+
+The value is a comma separated list of CIDRs. The allowed client IP source ranges can be specified through the annotation.
+
```

Comment:
Also add note as Adding an annotation to an Ingress rule overrides any global restriction